File: BRaVe-Mobile-Backend/Models/SetPinRequest.cs
namespace BRaVe_Mobile_Backend.Models
{
    public class SetPinRequest
    {
        public string DeviceId { get; set; }   
        public string? RequestIp { get; set; }   
        public int TenantId { get; set; }
        public string Code { get; set; }
        public byte[] OldPin { get; set; }
        //public bool IsDefault { get; set; }
        public byte[] NewPin { get; set; }     
    }
}

File: BRaVeMobile/app/src/main/java/intl/iom/bravemobile/models/SetPinRequest.java
package intl.iom.bravemobile.models;

public class SetPinRequest {

    public String code;
    public String oldPin ;
    public String newPin ;
    public String confirmedPin ;

    public SetPinRequest(String code, String oldPin, String newPin) {
        this.code = code;
        this.oldPin = oldPin;
        this.newPin = newPin;
    }

    public SetPinRequest(String code, String oldPin, String newPin, String confirmedPin) {
        this.code = code;
        this.oldPin = oldPin;
        this.newPin = newPin;
        this.confirmedPin = confirmedPin;
    }
}
